### PR TITLE
refactor(docs): update README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by jpetto**
# Goal

clean up docs a bit.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,10 +26,42 @@ If you've found a bug, or have a feature idea that you you'd like to see, follow
...
+
+If you see a bug that is not yet assigned to anyone, start a conversation with an engineer in the ticket itself, expressing your interest in taking the bug. If you take the bug, someone will set the ticket to Assigned to Contributor, so we can be proactive about helping you succeed in fixing the bug.
 
 When you have some code written, you can open up a Pull Request, get your code reviewed, and see your code merged into the codebase.
 
+### Commit format
+
+We use standard changelog and enforce [conventional commits](https://www.conventionalcommits.org/).
+
+Commits should be formated as `type(scope): message`
```
Comment (mmiermans) [inline CONTRIBUTING.md:44]: I think some examples would be useful here.

Have we documented anywhere what _scope_ should be? My understanding is that it's a freeform string, that's set to the name of the component that's being changed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,10 +26,42 @@ If you've found a bug, or have a feature idea that you you'd like to see, follow
...
+| docs | Documentation only changes |
+| style | Changes that do not affect the meaning of the code (white-space, formatting,missing semi-colons, etc) |
+| refactor | A code change that neither fixes a bug nor adds a feature |
+| perf | A code change that improves performance |
+| test | Adding missing or correcting existing tests |
+| chore | Changes to the build process or auxiliary tools and libraries such as documentation generation |
+
+### Releasing
+
+A new version is released when a merge or push to `main` occurs.
```
Comment (mmiermans) [inline CONTRIBUTING.md:61]:
```suggestion
A new version is released when a merge or push to `main` occurs that matches the commit format described above.
```

File: README.md [hunk 3]
```diff
@@ -42,7 +35,9 @@ The above will update any necessary snapshot files to be used on future test run
 
 As you can infer from the above, snapshots do not test the actual infrastructure result of running the synthesized terraform, meaning components should be tested manually (see below) to ensure they are performing the expected tasks prior to writing snapshot tests.
 
-### Testing in this repo
+### Testing in AWS
+
+While snapshot testing is great for things like regressions, it doesn't actually tell us if the code we've provided (e.g. the configuration of a particular AWS service) can be built in AWS.
```
Comment (mmiermans) [inline README.md:40]: Replace `npm run build && npm run synth` by `npm run build:dev` below?

**Review by mmiermans — APPROVED** [review]
Nice work! I added some suggestions, but this is a huge improvement.

**Review by chelseatroy — APPROVED** [review]
LGTM!

**Pull request merged**